lib/src/templates/big_fact_slide.dart

```diff
@@ -0,0 +1,83 @@
+import 'package:auto_size_text/auto_size_text.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_deck/flutter_deck.dart';
+import 'package:flutter_deck/src/flutter_deck_layout.dart';
+import 'package:flutter_deck/src/templates/templates.dart';
+
+/// A slide widget that represents a big fact Slide
+/// This class is used to create the big fact slide in a slide deck.
+/// It renders two lines of text, a Big title and a subtitle.
+/// It reders the default header and footer of the slide deck, if they are
+/// enabled in the configuration.
+class FlutterDeckBigFactSlide extends StatelessWidget {
+  /// Creates a new title slide.
+  /// The [title] and [subtitle] arguments must not be null.
+  /// The [backgroundBuilder] argument is optional.
+  /// The [subtitleMaxLines] is the maximum number of lines for the title by
+  /// default = 3.
+  const FlutterDeckBigFactSlide({
+    required this.title,
+    this.subtitle,
+    this.subtitleMaxLines,
+    this.backgroundBuilder,
+    super.key,
+  });
+
+  /// The title of the slide.
+  final String title;
+
+  /// The subtitle of the slide.
+  final String? subtitle;
+
+  /// maxLines for the subtitle of the slide.
+  final int? subtitleMaxLines;
```

Comment:
Maybe make this property non-nullable and assign the default value of 3 in the constructor?

Reply:
yes we can but then we will put the default value as 3 inside the named constructor `FlutterDeckSlide.bigFact`

then it will be like this 

` FlutterDeckSlide.bigFact({
    required String title,
    String? subtitle,
    WidgetBuilder? backgroundBuilder,
    int subtitleMaxLines = 3,
    FlutterDeckThemeData? theme,
    Key? key,
  }) : this._(
          builder: (context) => FlutterDeckBigFactSlide(
            title: title,
            subtitle: subtitle,
            backgroundBuilder: backgroundBuilder,
            subtitleMaxLines: subtitleMaxLines,
          ),
          theme: theme,
          key: key,
        );`

Reply:
Ahh, I see. Then we can do it like:

```dart
const FlutterDeckBigFactSlide({
  required this.title,
  this.subtitle,
  int? subtitleMaxLines,
  this.backgroundBuilder,
  super.key,
}) : subtitleMaxLines = subtitleMaxLines ?? 3;
```

Reply:
yuppp